docs/examples/media/podcasts-media.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: Podcasts
+---
+
+import PodcastsList from "@site/src/components/podcasts-list";
+
+:::info
+For an archive of pre-v10 content, see the [Cypress v9 Podcast](http://v9.docs.cypress.io/examples/media/podcasts-media) page.
+:::
+
+<!--
+  If you're looking to add content to our Podcasts page,
+  Go to src/data/podcasts.json and add your podcast.
+-->
+
+<!-- <PodcastsList /> -->
```

Comment:
Should this be commented out?

Reply:
Ah, yes. These are in flight until we work out the details with the Ecosystem team. This is because there's no v10 specific content here and this interim state is to show the pages with only the v9 callout at the top. I'll shift this PR to WIP to avoid confusion.